src/components/BookCard.js

```diff
@@ -1,22 +1,38 @@
 import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from '../redux/features/book/bookSlice';
 import '../styles/bookCard.css';
```

Comment:
Nice job fixing the issues pointed out by the previous reviewer✨. However, When I try to run your app the following error messages are displayed:

<img width="948" alt="image" src="https://github.com/kwesiObuobi/bookstore/assets/37116322/327aa42d-74ba-4aab-b059-66afcc711047">

Please make sure that your imports are correctly capitalized🙏

Reply:
Wow. Those files just keep renaming themselves, @Shisui6. Let me try one more time.